package.json

```diff
@@ -40,7 +40,7 @@
     "@types/color": "^3.0.2",
     "@types/js-cookie": "^3.0.0",
     "@types/node": "^14.10.1",
-    "@types/react": "^16.9.49",
+    "@types/react": "^17.0.37",
```

Comment:
Bonus: Solves some typing error that sprung up in VSCode just now. (non-functional change)

Reply:
shouldn't the types versions match our react/react-dom versions? i had to downgrade @types/react-dom to 16.x to get rid of vscode errors

Reply:
Though it is not necessary in this case, that's also an option here... For sure. 

This is a safe change, as the types for 17 are backwards compatible to 16 (only features added, nothing has been deprecated) but you're right, I'll downgrade `@types/react-dom` instead. 👍

Reply:
oh so maybe the errors i was getting was due to the mismatch just in the types version. but yeah, i'd prefer to match our lib versions if the result is the same. :)

Reply:
"totes!", that was it... and done. 
v16 is on: the minors/patches don't need to match the libraries, because those advance as they fix and work on the @types packages themselves.